Database/Buffer/Backend/index.ts

```diff
@@ -48,10 +52,9 @@ export class Backend {
 	}
 
 	async alarm(): Promise<void> {
-		const archiveTime = 20 * 1000 //TODO: after settlement
 		const now = Date.now()
 		const archiveThreshold = isoly.DateTime.truncate(
-			isoly.DateTime.create(now - archiveTime, "milliseconds"),
+			isoly.DateTime.create(now - this.archiveTime, "milliseconds"),
```

Comment:
this.archiveTime is in seconds and create() takes miliseconds and now is also in miliseconds.